code/modules/surgery/bodyparts/bodyparts.dm

```diff
@@ -0,0 +1,337 @@
+
+//to update the bodypart's icon when not attached to a mob
+/obj/item/bodypart/proc/update_icon_dropped()
+	overlays.Cut()
+	var/image/I = get_limb_icon(1)
+	if(I)
+		I.pixel_x = px_x
+		I.pixel_y = px_y
+		overlays += I
+
+//Gives you a proper icon appearance for the dismembered limb
+/obj/item/bodypart/proc/get_limb_icon(dropped)
+	var/image/I
+
+	var/icon_gender = (body_gender == FEMALE) ? "f" : "m" //gender of the icon, if applicable
+
+	if((body_zone != "head" && body_zone != "chest"))
+		should_draw_gender = FALSE
+
+	var/image_dir
+	if(dropped)
+		image_dir = SOUTH
+	if(status == ORGAN_ORGANIC)
+		if(should_draw_greyscale)
+			if(should_draw_gender)
+				I = image("icon"='icons/mob/human_parts_greyscale.dmi', "icon_state"="[species_id]_[body_zone]_[icon_gender]_s", "layer"=-BODYPARTS_LAYER, "dir"=image_dir)
+			else
+				I = image("icon"='icons/mob/human_parts_greyscale.dmi', "icon_state"="[species_id]_[body_zone]_s", "layer"=-BODYPARTS_LAYER, "dir"=image_dir)
+		else
+			if(should_draw_gender)
+				I = image("icon"='icons/mob/human_parts.dmi', "icon_state"="[species_id]_[body_zone]_[icon_gender]_s", "layer"=-BODYPARTS_LAYER, "dir"=image_dir)
+			else
+				I = image("icon"='icons/mob/human_parts.dmi', "icon_state"="[species_id]_[body_zone]_s", "layer"=-BODYPARTS_LAYER, "dir"=image_dir)
+	else
+		if(should_draw_gender)
+			I = image("icon"='icons/mob/augments.dmi', "icon_state"="[body_zone]_[icon_gender]_s", "layer"=-BODYPARTS_LAYER, "dir"=image_dir)
+		else
+			I = image("icon"='icons/mob/augments.dmi', "icon_state"="[body_zone]_s", "layer"=-BODYPARTS_LAYER, "dir"=image_dir)
+		return I
+
+
+	if(!should_draw_greyscale)
+		return I
+
+	//Greyscale Colouring
+	var/draw_color
+
+	if(skin_tone) //Limb has skin color variable defined, use it
+		draw_color = skintone2hex(skin_tone)
+	if(species_color)
+		draw_color = species_color
+	if(mutation_color)
+		draw_color = mutation_color
+
+	if(draw_color)
+		I.color = "#[draw_color]"
+	//End Greyscale Colouring
+
+	return I
+
+
+
+/obj/item/bodypart/chest
+	name = "chest"
+	desc = "why is it detached..."
+	icon_state = "chest"
+	max_damage = 200
+	body_zone = "chest"
+	body_part = CHEST
+	px_x = 0
+	px_y = 0
+	var/obj/item/cavity_item
+
+/obj/item/bodypart/chest/Destroy()
+	if(cavity_item)
+		qdel(cavity_item)
+	return ..()
+
+/obj/item/bodypart/l_arm
+	name = "left arm"
+	desc = "why is it detached..."
+	icon_state = "l_arm"
+	max_damage = 75
+	body_zone ="l_arm"
+	body_part = ARM_LEFT
+	px_x = -6
+	px_y = 0
+
+/obj/item/bodypart/r_arm
+	name = "right arm"
+	desc = "why is it detached..."
+	icon_state = "r_arm"
+	max_damage = 75
+	body_zone = "r_arm"
+	body_part = ARM_RIGHT
+	px_x = 6
+	px_y = 0
+
+/obj/item/bodypart/l_leg
+	name = "left leg"
+	desc = "why is it detached..."
+	icon_state = "l_leg"
+	max_damage = 75
+	body_zone = "l_leg"
+	body_part = LEG_LEFT
+	px_x = -2
+	px_y = 12
+
+/obj/item/bodypart/r_leg
+	name = "right leg"
+	desc = "why is it detached..."
```

Comment:
Replace with "You put your right leg in, your right leg out. In, out, in, out, shake it all about. And apparently then it detaches. The hokey cokey has certainly changed a lot since space colonisation."